src/api/verification_library.py

```diff
@@ -100,3 +100,25 @@ def get_library_item(self, item_name: str) -> Dict:
         }
 
         return item_dict
+
+    def get_library_items(self, items: List = []) -> Union[List, None]:
+        """Get the json definition and meta information of a list of specific library items.
+
+        Args:
+            items:  list of str, default []. Library items to get. By default, get all library items loaded at instantiation.
+
+        Returns:
+            Dict that contains summary information of library items.
+        """
+
+        # check `items` arg type
+        if not isinstance(items, List):
+            logging.error(f"items' type must be List. It can't be {type(items)}.")
+            return None
+
+        # get each lib item's summary info
+        item_list = []
```

Comment:
By default, get all library items loaded at instantiation. this logic needs to be added here. please also add a test for this.

Reply:
I'm not sure if I understood this point. Could you be more specific? 

From my understanding, the goal of the ```get_library_items``` method is to grab multiple libraries' info. And we can iterate the ```get_library_item``` method to get this job done.

Reply:
Addressed.